sqlalchemy_cockroachdb/__init__.py

```diff
@@ -1,7 +1,7 @@
 from sqlalchemy.dialects import registry as _registry
 from .transaction import run_transaction  # noqa
 
-__version__ = "1.4.0.dev0"
+__version__ = "1.4.0"
```

Comment:
Yup, that's all we should need to do. (Well that, and remember to bump the version to "1.4.1.dev0" after the release gets pushed to PyPI.)